userguide/tasks.rst

```diff
@@ -889,16 +889,23 @@ To choose a dataset, click |ui-browse| and select the dataset from the
 expandable tree. Multiple :guilabel:`Source Datasets` can be chosen.
 
 Start by selecting the :guilabel:`Source Datasets` to be replicated.
-Source datasets on a remote system need a
-:ref:`Periodic Snapshot Task <Periodic Snapshot Tasks>`, or the
-snapshots can be manually selected by setting
-:guilabel:`Replicate Custom Snapshots` and entering a snapshot
-:guilabel:`Naming Schema`. The schema is a pattern of the name and
-`strftime(3) <https://www.freebsd.org/cgi/man.cgi?query=strftime>`__
+Source datasets on the local system are replicated using existing
```

Comment:
The deleted section mentions manually choosing snapshots.  Is that removed or covered elsewhere?

Reply:
It's a split of the behavior between local source datasets and remote source datasets. Remote source text is correct, but I need to port the "Replicate Custom Snapshots" text to the new local source datasets paragraph